scripts/zones/Yahse_Hunting_Grounds/npcs/Yahse_Wildflower.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- Area: Yahse Hunting Grounds
+--  NPC: Yahse Wildflower
+-- Involved in quest Children of the Rune
+-- pos 370.6285 0.6692 153.3728
+-----------------------------------
+local ID = require("scripts/zones/Yahse_Hunting_Grounds/IDs")
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+  -- CHILDREN OF THE RUNE
+  local childrenOfTheRune = player:getQuestStatus(ADOULIN, tpz.quest.id.adoulin.CHILDREN_OF_THE_RUNE)
+  local yahseWildflowerPetalKeyItem = tpz.ki.YAHSE_WILDFLOWER_PETAL
+  if childrenOfTheRune == QUEST_ACCEPTED and not player:hasKeyItem(yahseWildflowerPetalKeyItem) then
+    player:addKeyItem(yahseWildflowerPetalKeyItem)
+    player:messageSpecial(ID.text.KEYITEM_OBTAINED, yahseWildflowerPetalKeyItem)
```

Comment:
[`npcUtil.giveKeyItem` will reduce these two lines to one call](https://github.com/project-topaz/topaz/blob/release/scripts/globals/npc_util.lua#L285-L293)

Reply:
Done. Much cleaner.

Had no idea this existed. Thanks for the pointer!